src/Components/App/App.js

```diff
@@ -97,6 +98,14 @@ class App extends Component {
         <Route exact path='/login' render={() => <Form /> } />
         <Route exact path='/' render={() => <MoviesContainer selectMovie={this.selectMovie} className='movie_container' signOutUser={this.signOutUser} removeFavourite={this.removeFavourite} handleFavourite={this.handleFavourite}/> } />
         <Route exact path='/favorites' render={() => <FavouritesContainer handleFavourite={this.handleFavourite}/>} />
+        <Route exact path='/movies/:id' render={({ match }) => {
```

Comment:
This ({ match }) looks interesting, haven't seen it before i don't think. It'd be great if you could explain this to us! If it works that is great!

Reply:
@SamuelColeman i think you can let me know through here what this does, before I merge it